clients/admin-ui/src/features/taxonomy/TaxonomyFormBase.tsx

```diff
@@ -59,27 +55,9 @@ const TaxonomyFormBase = ({
 
   const handleSubmit = async (newValues: FormValues) => {
     setFormError(null);
-    // parent_key and fides_keys are immutable
-    // parent_key also needs to be undefined, not an empty string, if there is no parent element
-    let payload: TaxonomyEntity;
```

Comment:
This payload mutation that was mandatory for every resource type is now included in the hooks.